### DISCUSSION TITLE
How to jump page in saga

### DISCUSSION THREAD

**Discussion by Statfine** [redux-saga/redux-saga#2318] [General]
### Description
After login success i want jump to other page, how can i jump page in saga

react-router-dom v6.3.0
redux-saga v1.1.3
react-redux v7.2.6
@reduxjs/toolkit v1.7.1

**Comment by neurosnap** [comment]
Hi!  This is an interesting topic because in general the recommendation is to **not** have redux and react-router mix.  Instead you would want to send a signal to your login component that informs it to redirect to the next page.  So the react component itself would check for a login success flag in your redux store and then you would `navigate` to the next page using react-router.

It's hard to go into more details without seeing some code, which I would be happy to look at.
